resources/zh/cemu.po

```diff
@@ -173,6 +173,30 @@ msgstr "繁体中文"
 msgid "unknown"
 msgstr "未知"
 
+#: CemuApp.cpp
+msgid "AccountId missing (The account is not connected to a NNID)"
+msgstr "帐户 ID 丢失 (帐户未连接到 NNID)"
```

Comment:
I think that `AccountId` should be treated in the same way as `IsPasswordCacheEnabled` and `AccountPasswordCache` below (and therefore should not be translated)